app/src/main/java/org/schabi/newpipe/util/StreamDialogDefaultEntry.java

```
@@ -0,0 +1,165 @@
+                        NewPipeDatabase.getInstance(fragment.requireContext()).streamDAO()
+                                .setUploaderUrl(serviceId, url, result.getUploaderUrl())
+                                .subscribeOn(Schedulers.io()).subscribe();
+                        openChannelFragment(fragment, item, result.getUploaderUrl());
+                    }, throwable -> ErrorUtil.openActivity(
+                            fragment.requireContext(),
+                            new ErrorInfo(
+                                    throwable,
+                                    UserAction.REQUESTED_CHANNEL,
+                                    url,
+                                    serviceId
+                            )));
+        } else {
+            openChannelFragment(fragment, item, item.getUploaderUrl());
+        }
+    }),
+
+    /**
+     * Enqueues the stream automatically to the current PlayerType.<br>
+     * <br>
+     * Info: Add this entry within showStreamDialog.
+     */
+    ENQUEUE(R.string.enqueue_stream, (fragment, item) ->
+        NavigationHelper.enqueueOnPlayer(fragment.getContext(), new SinglePlayQueue(item))
+    ),
+
+    ENQUEUE_NEXT(R.string.enqueue_next_stream, (fragment, item) ->
+        NavigationHelper.enqueueNextOnPlayer(fragment.getContext(), new SinglePlayQueue(item))
+    ),
+
+    START_HERE_ON_BACKGROUND(R.string.start_here_on_background, (fragment, item) ->
+            NavigationHelper.playOnBackgroundPlayer(fragment.getContext(),
+                    new SinglePlayQueue(item), true)),
+
+    START_HERE_ON_POPUP(R.string.start_here_on_popup, (fragment, item) ->
+            NavigationHelper.playOnPopupPlayer(fragment.getContext(),
+                    new SinglePlayQueue(item), true)),
+
+    SET_AS_PLAYLIST_THUMBNAIL(R.string.set_as_playlist_thumbnail, (fragment, item) -> {
+    }), // has to be set manually
+
+    DELETE(R.string.delete, (fragment, item) -> {
+    }), // has to be set manually
```

These should throw `UnsupportedOperationException`